src/components/FooterContainer/FooterContainer.tsx

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2019 - 2022 Gemeente Amsterdam
+import type { FC } from 'react'
+import {
+  Link,
+  Footer as StyledFooter,
```

Comment:
The naming confuses me, because usually StyledComponent refers to an Amsterdam Styled Component that receives additional styling. Here, the Styled refers to a plain ASComponent.

Reply:
Ah oke! That makes sense. It is just that this component itself becomes the actual Footer component in our application, therefore I wanted the name for that;) But I'll revert it.